doc/url_parameters.md

```diff
@@ -0,0 +1,22 @@
+# URL Parameters
+
+| Parameter | Value | Description |
+| --------- | ----- | ----------- |
+| `p` | `geographic`, `arctic` or `antarctic` | Selected projection. |
+| `v` | `[minX,minY,maxX,maxY]` | Extent of the map viewport in units based on the projection selected (degrees for EPSG:4326, meters for others). |
+| `l` | `[layer_id1],[layer_id2],[layer_id3]` | Active layer list where `layer_id` is the identifier of the layer as defined in the configuration file. Any number of baselayers or overlays may be specified (separated by a `,`). |
+| `l=[layer_id](hidden)` | `(hidden)` | If present, toggles the visibility of the layer as identified by the `layer_id` in the configuration file. |
```

Comment:
how about "if present, this layer is appears in the layer list but is not shown on the map" or similar wording that sounds good.